features/lwipstack/lwipopts.h

```diff
@@ -111,7 +111,11 @@
 #define TCPIP_THREAD_STACKSIZE      LWIP_ALIGN_UP(MBED_CONF_LWIP_TCPIP_THREAD_STACKSIZE, 8)
 #endif
 
+#ifdef MBED_CONF_LWIP_TCPIP_THREAD_PRIORITY
```

Comment:
Could we add a comment with the default value.

Reply:
Actually, I'd kind of prefer not to have a default value here. This isn't a case where we need `null` in the JSON to mean "do something sensible".

We could just put the `"osPriorityNormal"` in the default JSON file, and have no `#ifdef` here.

Reply:
Yes I am fine with "osPriorityNormal" in the default mbed_lib.json, with no #ifdef in header file.